manifests/charts/istio-cni/templates/resourcequota.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.cni.resourceQuotas.enabled }}
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: istio-cni-resource-quota
+  namespace: {{ .Release.Namespace }}
+spec:
+  hard:
+    pods: {{ .Values.cni.resourceQuotas.pods | quote }}
+  scopeSelector:
+    matchExpressions:
+    - operator: In
+      scopeName: PriorityClass
+      values:
+      - system-node-critical
```

Comment:
should be based on values.priorityClassName?

Reply:
This priority class targets only cni, and We hard coded CNI priority class to be `system-node-critical`: https://github.com/istio/istio/blob/bb280f568f4f5369f3732130cd1bd8481b5fdd3c/manifests/charts/istio-cni/templates/daemonset.yaml#L55